### PR TITLE
{bio}[intel/2017b] MIRA v4.0.2 (stable) + v4.9.6 (development)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 3 (3 easyconfigs in this PR)
node2499.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f822c8c14c4d01af6c1645425ca26efe for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2499.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c9598e7a4e2bdf82b5be42632268ae20 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2088.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/832a51695d312b2428e93f2c340d334f for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 5 out of 5 (6 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/0028ce3065a1d4828e5b39c0ad6e4bbe for a full test report.

File: easybuild/easyconfigs/m/MIRA/MIRA-4.0.2-intel-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'MIRA'
+version = '4.0.2'
+
+homepage = 'https://sourceforge.net/projects/mira-assembler/'
+description = """MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina),
+ IonTorrent data and PacBio (the later at the moment only CCS and error-corrected CLR reads)."""
```
Comment (verdurin) [inline easybuild/easyconfigs/m/MIRA/MIRA-4.0.2-intel-2017b.eb]: @boegel - I know you've copied and pasted here, but surely this should be `the latter`.

File: easybuild/easyconfigs/m/MIRA/MIRA-4.9.6-intel-2017b.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'MIRA'
+version = '4.9.6'
+
+homepage = 'https://sourceforge.net/projects/mira-assembler/'
+description = """MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina),
+ IonTorrent data and PacBio (the later at the moment only CCS and error-corrected CLR reads)."""
```
Comment (verdurin) [inline easybuild/easyconfigs/m/MIRA/MIRA-4.9.6-intel-2017b.eb]: @boegel same here.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**